### DISCUSSION TITLE
XMLSec 1.3.2-rc1

### DISCUSSION THREAD

**Discussion by lsh123** [lsh123/xmlsec#712] [Announcements]
The XML Security Library 1.3. release includes the following changes:

- (xmlsec-openssl) Fixed padding for GOST 2001 and 2012 signatures.
- (xmlsec-nss) Added support for reading PEM certificates.
- (xmlsec-nss) Added a check to ensure that the key certificat matches the key.
- (xmlsec-nss) Added support for xmlsec command line tool '--verify-keys' option.
- (xmlsec-gnutls) Added support for GOST R 34.11-94, GOST R 34.11-2012 256 bit, and GOST R 34.11-2012 512 bit digest algorithms.
- (xmlsec-gnutls) Added support for GOST R 34.10-2001, GOST R 34.11-2012 256 bit, and GOST R 34.11-2012 512 bit signature algorithms.
- (xmlsec-gnutls) Added support for xmlsec command line tool '--verify-keys' option.
- (xmlsec-gnutls) Added check to ensure that the key certificat matches the key.
- (xmlsec-mscng) Added support for xmlsec command line tool '--verify-keys' option.
- (xmlsec-mscng) Replaced windows.h includes with wincrypt.h includes where possible.
- (xmlsec-mscrypto) Replaced windows.h includes with wincrypt.h includes where possible.
- (xmlsec command line tool) Added `--base64-line-size` option to control the base64 encoding line size.
- (MSVC build) Added 'ftp' and 'http' options to control FTP and HTTP support. FTP support is disabled by default.
- (MinGW build) The xmlsec-mscrypto is moved down in the default crypto library selection list as it is now in maintanance mode (use "--with-default-crypto" option to force the selection).
- (MinGW build) Fixed the static libraries build with "--enable-static-linking" option.
- Several other small fixes (see [more details](https://github.com/lsh123/xmlsec/commits/master)).

Please test the release [candidate](https://www.aleksey.com/xmlsec/download/xmlsec1-1.3.2-rc1.tar.gz) ([signature](https://www.aleksey.com/xmlsec/download/xmlsec1-1.3.2-rc1.sig)) and let me know if you see any issues!

<hr /><em>This discussion was created from the release <a href='https://github.com/lsh123/xmlsec/releases/tag/xmlsec_1_3_2_rc1'>XMLSec 1.3.2-rc1</a>.</em>

**Comment by vmiklos** [comment]
Thanks, looks good on the odfsig side.

**Reply by lsh123** [reply]
thanks for testing!
